README.md

```
@@ -1,31 +1,26 @@
 
 ## 🌟 Servers
 
 These servers aim to demonstrate MCP features and the official SDKs.
 
-- **[Deep Thinking](src/deep-thinking)** - Dynamic and reflective
-  problem-solving through thought sequences.
-- **[Tasks](src/tasks)** - Self guided task planning, management, execution and
-  completion.
-- **[Knowledge Graph Memory](src/knowledge-graph-memory)** - Persistent memory
-  for through a local knowledge graph.
+- **[Deep Thinking](src/deep-thinking)** - Dynamic and reflective problem-solving through thought sequences.
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 80; Actual: 108

(MD013, line-length)